pyproject.toml

```diff
@@ -4,21 +4,6 @@ requires = ["setuptools", "wheel"]
 # https://github.com/python-versioneer/python-versioneer/issues/193
 # build-backend = "setuptools.build_meta"
 
-[tool.black]
-target-version = ["py38", "py39", "py310"]
-extend_excludes = '''
-(
-     docs/.*
-    | tests/.*.fixture
-    | .*.ge_store_backend_id
-)
```

Comment:
These were previously excluded from formatting.
Do we still need to exclude them?

Reply:
I think we do since these all look like specially formatted files. For example, changing `docs/.*` may change how things get rendered in our docs.

Reply:
Actually it looks like most of these where already being ignored by `ruff lint` so they are already being ignored by `ruff format`.
There doesn't appear to be a way to have a separate `format` ignore list.